ogre_mesh_viewer.py

```diff
@@ -53,6 +57,18 @@ def config_option_combo(rs, option):
 def printable(str):
     return str.encode("utf-8", "replace").decode()
 
+def open_file_dialog():
+    root = tk.Tk()
+    root.withdraw()
+    infile = filedialog.askopenfilename(
+        title="Open",
+        filetypes=[
+            ("OGRE Mesh files", "*.mesh"),
+            ("OGRE Scene files", "*.scene"),
+            ("ASSIMP Supported", "*.fbx,*.dae,*.gltf,*.glb,*.obj"),
```

Comment:
it seems you cannot specify a list as a filter. It does not show me any .obj files wheres the one below does

Reply:
The separator should be a space, that was the problem